### ISSUE TITLE
Add support for marking down Smart Chips 

### ISSUE THREAD

**Issue by jshank** [evbacher/gd2md-html#99]
Enhance the tool to handle Google [Smart Chips](https://support.google.com/docs/answer/10710316?hl=en) such as dates and email addresses. These items are currently ignored by gd2md leaving blank spaces where any Smart Chips are in the document.

**Comment by mg0716** [comment]
This is especially useful given the new Google Calendar → Google Doc auto doc generation that fills in a meeting name and attendees as chips.

**Comment by chrizbo** [comment]
Yes, I use smart chips in my Google Doc headings all the time for people and dates. I'd love to vote to get this updated!
